cves/CVE-2002-0392.yml

```diff
@@ -31,7 +31,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  This vulnerability was caused by a lack of error-checking which allowed HTTPD
+  to accept an invalid size, potentially causing a denial-of-service
```

Comment:
An invalid size of what? Like some sort of size parameter or something?

Reply:
I didn't really find a clear explanation for what the security bulletin meant by "invalid size," but from context it seems to be some sort of memory allocation issue.